conf/logback.xml

```diff
@@ -45,6 +45,7 @@
 
     <logger name="com.google.inject" level="INFO"/>
     <logger name="org.asynchttpclient.netty.channel.DefaultChannelPool" level="INFO"/>
+    <logger name="application" level="DEBUG"/>
```

Comment:
I am not sure there is a need for it as the root level has already debug level here.